src/BikeDataProject.API/connection_string.txt

```diff
@@ -1 +1 @@
-Host=125.0.0.1;Port=5433;Database=bikedata;Username=postgres;Password=mixbeton
\ No newline at end of file
+Host=127.0.0.1;Port=5433;Database=bikedata;Username=postgres;Password=mixbeton
```

Comment:
<img src="https://www.boels.be/assets/modules/products/concrete_mixer_140_l_big_1.jpg"/>